benchmarks/README.md

```diff
@@ -0,0 +1,15 @@
+## Benchmarks
+
+Benchmarks instrumenting an OpenCensus enabled gRPC backend vs one that
+isn't. Once tests are run it generates some charts of:
+
+* allocs/op vs QPS
+* throughput/op vs QPS
+
+
+### Running it
+```shell
+make all
```

Comment:
$ make all

to distinguish it is a common.

Reply:
What does `to distinguish it is a common.` mean?
I used `make all` as in the past I've had people complain that they couldn't copy and paste, but I'll make the `$ make all` update :)